scripts/sample_maps.py

```diff
@@ -62,22 +73,23 @@ def log_gaussian_prior_b(map_data, sigma):
 
 gaussian_prior_score_b = jax.vmap(jax.grad(log_gaussian_prior_b), in_axes=[0,0])
 
-def log_likelihood(x, sigma, meas_shear, sigma_mask):
-  """ Likelihood function at the level of the measured shear
-  """
-  x = x.reshape((360, 360,2))
-  ke = x[...,0]
-  kb = x[...,1]
-  #print(sigma)
-  #sigma = sigma[0]
-  model_shear = jnp.stack(ks93inv(ke, kb), axis=-1)
-
-  return - jnp.sum((model_shear - meas_shear)**2/((FLAGS.sigma_gamma)**2 + sigma**2 + sigma_mask) )/2.
+def main(_):
 
-likelihood_score = jax.vmap(jax.grad(log_likelihood), in_axes=[0,0, None, None])
+  std1 = jnp.expand_dims(fits.getdata(FLAGS.std1).astype('float32'), -1)
+  std2 = jnp.expand_dims(fits.getdata(FLAGS.std2).astype('float32'), -1)
+  sigma_gamma = jnp.concatenate([std1, std2], axis=-1)
 
+  def log_likelihood(x, sigma, meas_shear, sigma_mask):
+    """ Likelihood function at the level of the measured shear
+    """
+    x = x.reshape((360, 360,2))
+    ke = x[...,0]
+    kb = x[...,1]
+    model_shear = jnp.stack(ks93inv(ke, kb), axis=-1)
+  
+    return - jnp.sum((model_shear - meas_shear)**2/((sigma_gamma)**2 + sigma**2 + sigma_mask) )/2.
```

Comment:
Does std1 already contain a high variance of masked regions?

Reply:
`std1` contains zeros in masked regions. High variance is constained in `sigma_mask`